### DISCUSSION TITLE
Pybricks Firmware

### DISCUSSION THREAD

**Discussion by alexDonofrio25** [pybricks/support#1392] [Q&A]
Hello everyone, I have problem with the latest firmware and the stdout.buffer.write function that is not working anymore. Anyone knows how to fix this or knows how to rollback to old version of the firmware?

**Comment by dlech** [comment]
There were some changes in the 3.3.x firmware version, so if you were using a version older than that, you can have a look at https://pybricks.com/projects/tutorials/wireless/hub-to-device/pc-communication/ to see what needs to be changed.

If you don't want to mess with that and just use the older firmware, you can download it from the GitHub releases page. For example, the last 3.2.x version is at https://github.com/pybricks/pybricks-micropython/releases/tag/v3.2.3.
